core/src/main/scala/org/apache/spark/deploy/SparkSubmit.scala

```diff
@@ -200,6 +200,7 @@ object SparkSubmit {
       // Yarn cluster only
       OptionAssigner(args.name, YARN, CLUSTER, clOption = "--name"),
       OptionAssigner(args.driverMemory, YARN, CLUSTER, clOption = "--driver-memory"),
+      OptionAssigner(args.driverCores, YARN, CLUSTER, clOption = "--cores"),
```

Comment:
Ah, this confused me a bit. `SparkSubmitArguments` has this option as `--driver-cores`. Perhaps this should be the same (also above for standalone mode)?